include/aws/s3/s3_client.h

```diff
@@ -481,6 +497,15 @@ struct aws_s3_meta_request_options {
     /* The type of meta request we will be trying to accelerate. */
     enum aws_s3_meta_request_type type;
 
+    /**
+     * Optional.
+     * The S3 operation name (e.g. "CreateBucket").
+     * You should only set this when type is AWS_S3_META_REQUEST_TYPE_DEFAULT;
```

Comment:
`Only set this when` implies that it's an error to set this in other cases. However, for some workflows, it might be simpler to always set this, and we may or may not use it.
```suggestion
     * This will only be used when type is AWS_S3_META_REQUEST_TYPE_DEFAULT;
```